client/src/components/share/use-share.tsx

```diff
@@ -0,0 +1,36 @@
+import i18next from 'i18next';
```

Comment:
Can you move `use-share` hook into `index.tsx`, there isn't a complicate pieces in it, and moving them away provide hazard more than comfort.

Reply:
I can move to index.tsx, By doing that you are compromising on TDD unit tests and scalability of code, 

If i merged to index.tsx. then i can't write proper unit test cases, it redirects to code scalability reduced. if we start extend this more features in future.

Reply:
```suggestion
import { useTranslation } from "react-i18next";
```